utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -83,7 +83,9 @@ object OkHttpClientHelper {
      * Apply HTTP proxy settings from the "https_proxy" or "http_proxy" environment variables.
      */
     val applyProxySettingsFromEnv: OkHttpClient.Builder.() -> Unit = {
-        val proxyUrl = Os.env["https_proxy"] ?: Os.env["http_proxy"]
+        val proxyUrl = Os.env["https_proxy"]?.let { if (!it.startsWith("http")) "https://$it" else it }
```

Comment:
I'd prefer if 'if (!it.startsWith("http")) "https://$it" else it }' was not redundant, e.g.: Os.env["https_proxy"]?.let { setProtocol(it, "https") }

Reply:
Done.